src/device-registry/routes/v2/events-old.js

```diff
@@ -0,0 +1,705 @@
+const express = require("express");
+const router = express.Router();
+const eventController = require("@controllers/create-event");
```

Comment:
this is just the old definitions fo V2, kept here for reference as refactoring goes on. @123MwanjeMike / @uman95 / @wabinyai